src/data/weather.ts

```diff
@@ -115,18 +115,18 @@ export const getSecondaryWeatherAttribute = (
     return extrema;
   }
 
-  let value: number;
+  let value: string;
```

Comment:
why convert to string? Could we just use the Math.round()?

Reply:
We should keep some precision, so don't just use Math.round(), and since we will just print it, I guess converting to string is fine.